spec/unit/puppet/type/rabbitmq_policy_spec.rb

```diff
@@ -195,6 +195,19 @@
     end.to raise_error(Puppet::Error, %r{Invalid delivery-limit value.*future})
   end
 
+  it 'accepts and converts the initial-cluster-size value' do
+    definition = { 'initial-cluster-size' => '3' }
+    policy[:definition] = definition
+    expect(policy[:definition]['initial-cluster-size']).to eq(3)
+  end
+
+  it 'does not accept non-numeric initial-cluster-size value' do
+    definition = { 'initial-cluster-size' => 'impressive' }
+    expect do
+      policy[:definition] = definition
+    end.to raise_error(Puppet::Error, %r{Invalid initial-cluster-size value \'impressive})
```

Comment:
```suggestion
    end.to raise_error(Puppet::Error, %r{Invalid initial-cluster-size value 'impressive})
```

Sorry, my first suggestion works too, but I think the escape backslash is redundant inside `%r{`